kodilla-exception/src/main/java/com/kodilla/exception/test/FlightSearch.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+
+public class FlightSearch {
+
+    HashMap<String, Boolean> flights = new HashMap<>();
+
+    public void findFlight(Flight flight) throws RouteNotFoundException {
+        flights.put("Gdansk", true);
+        flights.put("Wroclaw", false);
+        flights.put("Krakow", true);
```

Comment:
Wiem ze to z tresci zadania, ale wiecej sensu ma dodawanie elementow w konstruktorze, zamiast robienie tego od nowa przy kazdym wywolaniu